test/controllers/main_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```

Comment:
I think this test file shouldn't exist. the `test` directory is usually used for minitest, a different testing framework from Rspec.